src/runtime/Makefile

```diff
@@ -150,6 +150,7 @@ FIRMWARETDVFVOLUMEPATH :=
 
 FIRMWARESEVPATH := $(PREFIXDEPS)/share/ovmf/OVMF.fd
 FIRMWARESNPPATH := $(PREFIXDEPS)/share/ovmf/AMDSEV.fd
+SNPCERTSPATH ?= /opt/snp/cert_chain.cert
```

Comment:
Is this something you'd like the users to have the power to overwrite via annotations?

Reply:
There should only be one certificate per node so setting this per pod shouldn't be necessary. Couldn't hurt, tho.